### PR TITLE
bitmaputil improvement

### PR DESCRIPTION
**Pull request by jDilshodbek**
The Android studio logcat is showing this warning during development  frequently while Using maplibre Android library:
`Drawable com.uznewmax.mytaxi:drawable/maplibre_user_icon_shadow has unresolved theme attributes! Consider using Resources.getDrawable(int, Theme) or Context.getDrawable(int). (Ask Gemini)
                                                                                                    java.lang.RuntimeException
                                                                                                    	at android.content.res.Resources.getDrawable(Resources.java:959)
                                                                                                    	at dev.b3nedikt.app_locale.AppLocaleResources.getDrawable(AppLocaleResources.kt:96)
                                                                                                    	at com.mapbox.mapboxsdk.utils.BitmapUtils.getDrawableFromRes(BitmapUtils.java:183)
                                                                                                    	at com.mapbox.mapboxsdk.utils.BitmapUtils.getDrawableFromRes(BitmapUtils.java:169)
                                                                                                    	at com.mapbox.mapboxsdk.location.LayerBitmapProvider.generateShadowBitmap(LayerBitmapProvider.java:30)
                                                                                                    	at com.mapbox.mapboxsdk.location.LocationLayerController.styleBitmaps(LocationLayerController.java:162)
                                                                                                    	at com.mapbox.mapboxsdk.location.LocationLayerController.applyStyle(LocationLayerController.java:98)
                                                                                                    	at com.mapbox.mapboxsdk.location.LocationComponent.applyStyle(LocationComponent.java:497)
                                                                                                    	at com.mapbox.mapboxsdk.location.LocationComponent.activateLocationComponent(LocationComponent.java:256)
                                                                                                    	at uz.mytaxi.map.maplibre.MapLibreImpl.setMyLocationIndicator(MapLibreImpl.kt:525)`
                                                                                                    	
As a solution, removed and updated some deprecated class methods.

Platform: Android OS
Version OS: Android 14
Device hardware : Xiaomi Redmi Note 13
Android skin: Harmony OS

### CODE REVIEW

File: platform/android/MapLibreAndroid/src/main/java/org/maplibre/android/utils/BitmapUtils.java [hunk 1]
```diff
@@ -180,7 +181,7 @@ public static Drawable getDrawableFromRes(@NonNull Context context, @DrawableRes
   @Nullable
   public static Drawable getDrawableFromRes(@NonNull Context context, @DrawableRes int drawableRes,
                                             @Nullable @ColorInt Integer tintColor) {
-    Drawable drawable = context.getDrawable(drawableRes);
+    Drawable drawable = ContextCompat.getDrawable(context,drawableRes);
```
Comment (louwers) [inline platform/android/MapLibreAndroid/src/main/java/org/maplibre/android/utils/BitmapUtils.java:184]:
```suggestion
    Drawable drawable = ContextCompat.getDrawable(context, drawableRes);
```

**Review by louwers — COMMENTED** [review]
Please run 

```
make android-check
```

locally, it's failing.

**Comment by jDilshodbek** [comment]
Should I run it in terminal? it is saying `make: *** No rule to make target `android-check'.  Stop.`
What lint tool is used to pass locally detect checks?

**Comment by louwers** [comment]
Yes it needs to be run in `platform/android` where the Makefile resides.

**Review by louwers — APPROVED** [review]
Thank you.

**Pull request merged**